catalog/app/containers/Bucket/requests/package.ts

```diff
@@ -206,53 +216,85 @@ export function useUpdatePackage() {
   )
 }
 
-const copyPackage = (
+const copyPackage = async (
   req: ApiRequest,
+  credentials: AWSCredentials,
   { message, meta, source, target, workflow }: CopyPackageParams,
   schema: JsonSchema, // TODO: should be already inside workflow
-) =>
-  uploadManifest(req, ENDPOINT_COPY, {
-    message,
-    meta: getMetaValue(meta, schema),
-    name: target.name,
-    parent: {
-      top_hash: source.revision,
-      registry: `s3://${source.bucket}`,
-      name: source.name,
+) => {
+  // refresh credentials and load if they are not loaded
+  await credentials.getPromise()
```

Comment:
Credentials aren't needed for Create and Upload package, for two of four requests. But I want to avoid repetition anyway in the next PR.
I was trying to make changeset as simple as possible to avoid mistakes